test/browser.test.js

```diff
@@ -8,6 +8,7 @@ var vm = require('vm');
 
 describe('browser support', function() {
   this.timeout(60000); // 60s to give browserify enough time to finish
+  sandbox.reset();
```

Comment:
This is wrong. It should be either `before(sandbox.reset)` or `beforeEach(sandbox.reset)`. I think beforeEach makes more sense.